crates/dbsp/src/operator/dynamic/input.rs

```diff
@@ -1577,12 +1577,18 @@ mod test {
         dbsp.kill().unwrap();
     }
 
+    /// This test is ignored because the asynchronous merger sometimes finishes
```

Comment:
there is no ignore?

Reply:
Oh, I think I fixed the problem (there was a real problem IIRC) and forgot to delete the comment. I'll push a fix.